include/dlaf/eigensolver/bt_band_to_tridiag/api.h

```diff
@@ -7,13 +7,41 @@
 // Please, refer to the LICENSE file in the root directory.
 // SPDX-License-Identifier: BSD-3-Clause
 //
+
 #pragma once
 
+#include "dlaf/matrix/matrix.h"
 #include "dlaf/types.h"
 
 namespace dlaf::eigensolver::internal {
 
-template <Backend backend, Device device, class T>
-struct BackTransformationT2B {};
+template <Backend B, Device D, class T>
+struct BackTransformationT2B {
+  static void call(const SizeType band_size, Matrix<T, D>& mat_e, Matrix<const T, Device::CPU>& mat_hh);
+};
+
+#ifdef DLAF_WITH_CUDA
+template <class T>
+struct BackTransformationT2B<Backend::GPU, Device::GPU, T> {
+  static void call(const SizeType band_size, Matrix<T, Device::GPU>& mat_e,
+                   Matrix<const T, Device::CPU>& mat_hh);
+};
+#endif
```

Comment:
Just curious, is this required so that you can define the "unimplemented" specialization for `Backend::GPU`?

Reply:
Yes, exactly. Not really needed, it was just because I extracted this PR from another branch, and they are somehow "innocuous leftovers".

In the next PR they are going to be removed/replaced.